charts/kueue/prometheus_values.yaml

```diff
@@ -0,0 +1,172 @@
+# Default values for kueue.
```

Comment:
Do we need this file? Could we parametrize the default Kueue values instead?

Reply:
Yea I'm not sure the best approach here.

We can provide custom TLS Configs and this provides an example. Or we maintain the TLS Configs for ServiceMonitors?

As of now the default Kueue values don't have anything close to this.

Reply:
I don't think we want to afford the duplcation. 

Could we just add commented-out section for the ServiceMonitor. And add a comment uncomment when using external certs? We do similarly for webhooks.

Reply:
Also, the name of the file is misleading `charts/kueue/prometheus_values.yaml`. It suggests enabling prometheus, but in fact it enables external certs for webhooks and metrics.

Reply:
Thats a pattern we follow for kustomize but for helm we aim to programmatic handle that.

I was thinking `prometheus_values` could be used as an example in documentation if one wanted this?

Reply:
> Thats a pattern we follow for kustomize but for helm we aim to programmatic handle that.

Hm, I see for helm we also use the pattern with comments, for example [here](https://github.com/kubernetes-sigs/kueue/blob/main/charts/kueue/values.yaml#L117-L133).

> I was thinking prometheus_values could be used as an example in documentation if one wanted this?

I would prefer to avoid the copy-paste duplication. I'm open to other suggestions how to do it than comments, but copy-pasting values.yaml is not maintainable imo.

Reply:
I removed this file.

We followed off offline and decided to go with UX as a followup. In this case we would maybe consider a user guide or a documentation update mentioning how one could enable the use case I want.

For now I will get this PR functionally without changing the defaults.